src/olympia/shelves/tests/test_serializers.py

```diff
@@ -98,7 +98,8 @@ def setUpTestData(cls):
     def setUp(self):
         self.search_pop_thm = Shelf.objects.create(
             title='Populâr themes',
-            endpoint='search-themes',
+            endpoint='search',
+            addon_type=10,
```

Comment:
use the addon type constant when creating the `Shelf` object (and elsewhere in this file)